cmd/helm/root.go

```diff
@@ -71,6 +71,7 @@ Environment variables:
 | $HELM_KUBETLS_SERVER_NAME          | set the server name used to validate the Kubernetes API server certificate                                 |
 | $HELM_BURST_LIMIT                  | set the default burst limit in the case the server contains many CRDs (default 100, -1 to disable)         |
 | $HELM_QPS                          | set the Queries Per Second in cases where a high number of calls exceed the option for higher burst values |
+| $HELM_LINT_CONFIG_FILE			 | Set the location for the config file for linting															  |
```

Comment:
may not matter so much but the others all start with lowercase `set` vs title-cased `Set`